docs/Noise.ipynb

```diff
@@ -0,0 +1,467 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "# What Noise Should I Choose?"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## Introduction"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "In the examples up to now, we've been using `.noise()` to make our queries differentially private. However, there are different types of noise available including Laplacian noise and Gaussian noise. In this notebook, we will describe how the two noise distributions differ and how can one choose which is more appropriate for your particular analysis. \n",
+    "\n",
+    "The default `.noise()` by OpenDP chooses one of these distributions for you based on the limited information you've provided. However, with a better understanding of both distributions, you'll be able to specify the type of noise and its parameters for higher utility!"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## Laplace vs. Gaussian"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "The Laplace and Gaussian distributions are the two primary mechanisms to add noise to ensure differential privacy and preserve relationships for statistical analysis. The two distributions may appear similar at a first glance, but there are key differences in their use in differential privacy techniques. \n",
+    "\n",
+    "**1. Definition of Privacy:**\n",
+    " Differential privacy mechanisms using the Laplace distribution satisfy the pure ε-dp, whereas differentially private outputs using the Gaussian mechanism satisfy a more relaxed definition of differential privacy, ε-δ dp. Therefore, Laplace provides stronger privacy guarantees. \n",
```

Comment:
There is actually a notion of "strength" of privacy measures. I would put this more along the lines that pure DP is a stronger definition of privacy than approximate-DP, and since the gaussian mechanism cannot satisfy pure-DP, it's categorically different which can make it difficult to compare the two mechanisms fairly.

Not so relevant to the notebook, but more broadly, you can think of the strength of a privacy measure as a kind of partial ordering. All pure-DP guarantees can be converted into approx-DP guarantees, but not vice-versa. This is why pure-DP is "stronger" than approx-DP. [The following graph is not complete](https://mermaid.live/edit#pako:eNp1kLsKgzAUhl8lHOhWKWR0KEg7dgjpZuIQ9NgKaiQk9CK-e62JthaaKf8lfD_pIdcFQgxlrW_5VRlLTly2ZDxMMGcwI1G0J6l4Ho4s80E6WVxwbB_Vyku84F4IZnRZ1bgro-XtZkPYXA0YKpKuM_oeSLOMvonUI5dshaYr9k_n_xRGP0OCMxmMLlPTMGkxeNgBW2jQNKoqxq_r37EEe8UGJcTjtcBSudpKkO0wVpWz-vxoc4itcbgF1xXK4rFSF6Mabw4vpWVwFw), but gives a sense of the relationships between some of the common privacy measures.